geomesa-fs/geomesa-fs-storage/geomesa-fs-storage-parquet/src/main/scala/org/locationtech/geomesa/fs/storage/parquet/ParquetFileSystemStorage.scala

```diff
@@ -81,6 +84,29 @@ object ParquetFileSystemStorage {
       observer.write(f)
     }
     override def flush(): Unit = observer.flush()
-    override def close(): Unit = CloseQuietly(Seq(writer, observer)).foreach(e => throw e)
+    override def close(): Unit = {
+      CloseQuietly(Seq(writer, observer)).foreach(e => throw e)
+      if (FileValidationEnabled.get.toBoolean) {
+        validateParquetFile(file, logger)
+      }
+    }
+  }
+
+  def validateParquetFile(file: Path, logger: com.typesafe.scalalogging.Logger): Unit = {
```

Comment:
instead of passing in the logger, you can just make `object ParquetFileSystemStorage extends LazyLogging {` and that will put a logger in scope for the function.